artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -177,6 +181,18 @@
    // will be handled by the server's http server.
    private boolean httpUpgradeEnabled;
 
+   private boolean socksEnabled;
```

Comment:
proxyEnabled is a more intuite name... 

especially if you specify a connector with ?useSocket=true;proxyHost=;proxyPort;proxyUser;proxyPassword...

so.. better property names:

- proxyEnabled
- proxyUser
- proxyPassword
- proxyPort
- proxyHost